src/modules/list-commands.ts

```diff
@@ -262,37 +251,63 @@ const _parseOutputByRange = (
     return lines;
 };
 
-const _echoFormatted = (value: string, indent: number = 0) =>
+const _echoFormatted = (value: string, indent: number = 0): void =>
     Echo.message(`${" ".repeat(indent)}${_options.prefix}${value}`, false);
 
+const _execCliHelp = (command?: string): string | never => {
+    const helpCommand = ListCommands.cmd(command);
+    const { code, stderr, stdout } = shell.exec(helpCommand, { silent: true });
+
+    const coloredHelpCommand = Formatters.purple(helpCommand);
+    Echo.message(`Running ${coloredHelpCommand} for commands and options...`);
+
+    if (code !== 0 || StringUtils.hasValue(stderr)) {
+        const coloredError = Formatters.red(stderr);
+        Echo.error(`Failed to run ${coloredHelpCommand}:\n\n${coloredError}`);
```

Comment:
Since the `if code` check implies that there may not be a `stderr` value, should the error formatting handle a possibly empty `coloredError` value? Possibly replace the `stderr` message with a reference to the `code` value.

Reply:
Yeah, that's a good point! Nice catch.